deriva/transfer/download/processors/transform/pride_transform_processor

```diff
@@ -0,0 +1,90 @@
+import csv
+import collections
+from deriva.transfer.download.processors.transform.base_transform_processor import BaseTransformProcessor
+
+class PrideMTDExportTransformProcessor(BaseTransformProcessor):
```

Comment:
I would prefer a single class that can be parameterized on construction to operate in different modes rather than four separate classes with largely the same boilerplate code with only minor differences in output behavior. 

For example, create a "output_mode" parameter under `processor_params` that takes string values of "MTD", "CV", "FM", etc.  Read the value of this parameter in `self.kwargs` when the `process` function is invoked and then write the output lines accordingly. If you factor out the mode-specific logic into separate functions, you wont have to duplicate all of the common input/output file opening logic.